src/accountManagement/index.ts

```diff
@@ -15,5 +15,5 @@
 * limitations under the License.
 */
 
-import api = require("./accountManagementApi");
-module.exports = api.AccountManagementApi;
+import { AccountManagementApi } from "./accountManagementApi";
+export = AccountManagementApi;
```

Comment:
Would it be better if we used `export default AccountManagementApi` instead of `export = ...`?
I think `export default` is nice to use when exporting a single component from a file. It's also more consistent with what other libraries use out there.

Reply:
No, export default does something different

Reply:
True, and it would bring a few changes to a couple of files. Was just wondering about it 😄 

Trying to figure out more about `export = ...` vs ` export { }`, as I've only seen `export = ...` accompanied by `import = require()`

LGTM 👍

Reply:
`export =` means you export the defined object directly (like `module.exports =` in node) whereas `export {}` exports a new object containing your bits. `export default` creates an unwanted `default` reference on the export when transpiled to node.